package.json

```diff
@@ -9,6 +9,7 @@
     "build-clean": "npm run clean && npm run build",
     "test-simple": "nyc mocha \"./test/**/*.spec.js\"",
     "test": "npm run clean && npm run gardener && npm run test-simple",
+    "coverage": "npm run build-clean && nyc --reporter=html --reporter=text mocha",
```

Comment:
I probably should remove this line, but it's really useful for writing tests.
I don't believe it could break a gardener somehow.